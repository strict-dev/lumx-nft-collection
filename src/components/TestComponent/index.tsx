import * as S from './styles'

const TestComponent = () => (
  <S.Wrapper>
    <h1>TestComponent</h1>
  </S.Wrapper>
)

export default TestComponent
