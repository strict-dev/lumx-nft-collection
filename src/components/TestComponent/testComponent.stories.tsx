import { Story, Meta } from '@storybook/react'
import TestComponent from '.'
export default {
  title: 'TestComponent',
  component: TestComponent
} as Meta

export const Default: Story = () => <TestComponent />
