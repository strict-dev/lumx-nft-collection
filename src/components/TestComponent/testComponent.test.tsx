import { render, screen } from '@testing-library/react'
import TestComponent from '.'

describe('<TestComponent />', () => {
  it('should render the heading', () => {
    const { container } = render(<TestComponent />)

    expect(
      screen.getByRole('heading', { name: /TestComponent/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
