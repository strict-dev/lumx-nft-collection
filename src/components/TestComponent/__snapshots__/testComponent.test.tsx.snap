// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestComponent /> should render the heading 1`] = `
<div
  class=""
>
  <h1>
    TestComponent
  </h1>
</div>
`;
